docs/cluster_up_down.md

```diff
@@ -28,12 +35,79 @@ a URL to access the management console for your cluster.
 
 ## Getting Started
 
-### Mac OS X
+### Linux
+
+| WARNING |
+| ------- |
+| In some cases, networking for pods will not work for containers in your cluster, especially in Fedora 24. To fix this, run `$ sudo iptables -F` before running `oc cluster up` |
```

Comment:
explain that this will flush the iptables.  I don't like to run random sudo commands on my machine if i don't know what it's supposed to do :)